agent/util-scripts/pbench-tool-meister-start

```diff
@@ -290,45 +290,50 @@ def start_tms_via_ssh(
                 # Record the child pid to wait below.
                 tms[host] = {"pid": pid, "status": "forked"}
         else:
-            args[0] = host
-            args[4] = tm_param_key
-            ssh_args = base_args + args
-            logger.debug(
-                "6b. starting remote tool meister, ssh_path=%r ssh_args=%r",
-                ssh_path,
-                ssh_args,
-            )
+            logger.debug("6b. starting remote tool meister on %s", host)
             try:
-                pid = os.spawnv(os.P_NOWAIT, ssh_path, ssh_args)
+                template.start(host, tm_param_key=tm_param_key)
             except Exception:
                 logger.exception(
                     "failed to create a tool meister instance for host %s", host
                 )
                 tms[host] = {"status": "failed"}
             else:
                 # Record the child pid to wait below.
-                tms[host] = {"pid": pid, "status": "spawned"}
+                tms[host] = {"status": "spawned"}
```

Comment:
Maybe we also need to update the comment, since we are not recording the child pid here